Include/cpython/code.h

```diff
@@ -17,51 +19,75 @@ typedef struct _PyOpcache _PyOpcache;
 /* Bytecode object */
 struct PyCodeObject {
     PyObject_HEAD
-    int co_argcount;            /* #arguments, except *args */
-    int co_posonlyargcount;     /* #positional only arguments */
-    int co_kwonlyargcount;      /* #keyword only arguments */
-    int co_nlocals;             /* #local variables */
-    int co_stacksize;           /* #entries needed for evaluation stack */
+
+    /* Note that not all fields are used in either hash or comparisons. */
+
+    /* metadata */
+    PyObject *co_filename;      /* unicode (where it was loaded from) */
+    /* co_name is used in for both the hash and comparisions.  This is
+     * done to preserve the name and line number for tracebacks and
```

Comment:
This comment already existed.  I only moved it and tweaked it slightly.  Perhaps I misunderstood the original comment?

(I did just notice a typo: "in for".)

Reply:
But you changed the sense of the comment -- previously it had "The rest aren't used in either hash or comparisons, except for co_name, used in both." That made sense because all the fields above the comment are used in hash and comparisons, which includes co_firstlineno, and then co_name for some reason comes a little later. Maybe it's worth marking up the per-field comments with a mark indicating that it's used in comparisons, and explain the meaning of the mark in a comment preceding all fields?

Reply:
Ah, I'll clarify that then.